pkgs/top-level/all-packages.nix

```diff
@@ -37237,4 +37237,6 @@ with pkgs;
   swaysettings = callPackage ../applications/misc/swaysettings { };
 
   aitrack = libsForQt5.callPackage ../applications/misc/aitrack { };
+
+  nitch = callPackage ../tools/misc/nitch { };
```

Comment:
There's a tools section here (search for `### TOOLS`) - please use that one and keep the list vaguely alphabetical.

Reply:
Corrected